src/app/providers/ApplyForm.js

```diff
@@ -20,8 +20,8 @@ type Props = {|
   onClose: () => void,
 |}
 
-const linkTooltip = (title: string) => (
-  <Tooltip triggerText={title}>
+const linkTooltip = (title: string, required: boolean) => (
+  <Tooltip triggerText={<p>{title} { required ? <span className='polyRequiredRed'>*</span> : '' } </p>}>
```

Comment:
Use css `:after` selector with `content` variable to do this, so that you will only need to specify className for the <Field> component everywhere.
And I'm not sure that we really need this, looks like task slipped through without Stan approvement. Asked question there